app/src/main/java/com/example/filemanager/ListAdapter.kt

```diff
@@ -1,16 +1,25 @@
 package com.example.filemanager
 
+import android.content.Context
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import android.widget.ImageView
 import android.widget.TextView
+import androidx.core.content.ContextCompat
 import androidx.recyclerview.widget.RecyclerView
 
-internal class ListAdapter : RecyclerView.Adapter<ListAdapter.ViewHolder>() {
-    private var items = listOf<ListModel>()
+internal class ListAdapter(
+    private val context: Context,
+    private val list: ArrayList<ListModel>,
```

Comment:
`ArrayList` is a **Java** class and has been repkaced with `MutableList` in **Kotlin**. Please use `MutableList` instead.

Reply:
I replaced the ArrayList with MutableList

Reply:
Ok you can update your PR, I cant see it.